lib/charms/vault_k8s/v0/vault_autounseal.py

```diff
@@ -0,0 +1,518 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
```

Comment:
This lib can be shipped without the LICENSE file so I would add the license header instead. Same for the other libs

Reply:
We are making explicit the license for the lib files because when distributed through charmhub they are not downloaded with the LICENSE file:
```
# Copyright 2024 Canonical Ltd.
# Licensed under the Apache2.0. See LICENSE file in charm source for details.
```